packages/openapi-fetch/src/index.js

```diff
@@ -3,6 +3,8 @@ const DEFAULT_HEADERS = {
   "Content-Type": "application/json",
 };
 
+const PATH_PARAM_RE = /\{[^{}]+\}/g;
```

Comment:
Note (mostly for myself): I tested this simple RegEx approach, and a more low-level “parse-y” approach, and V8 seemed to favor this RegEx + `.match()` for performance. Before, this library used a static `.replace()`, and I wasn’t able to find a more performant alternative easily. There shouldn’t be any runtime impact (even splitting hairs).

Though I do value readability in code in general, when developing a library I think it is important to sweat the micro-optimizations, and not introduce runtime slowness just for the sake of readable code. I think that’s a luxury only afforded at the application level, not the library level.